File: src/main/java/com/micabytes/ink/StoryJson.java
package com.micabytes.ink;

import org.jetbrains.annotations.NonNls;

@SuppressWarnings("UtilityClass")
public final class StoryJson
{
  @NonNls public static final String FILE = "file";
  @NonNls public static final String FILES = "files";
  public static final String CONTENT = "content";
  public static final String ID = "id";
  public static final String COUNT = "count";
  public static final String VARIABLES = "variables";
  public static final String CONTAINER = "container";
  public static final String COUNTER = "counter";
  public static final String TEXT = "text";
  public static final String CHOICES = "choices";
  public static final String CHOICES_DETAIL = "choice";
  public static final String IMAGE = "image";
  public static final String RUNNING = "running";

  private StoryJson() {
    throw new AssertionError("StoryJson should never be initialized.");
  }
}

File: src/main/kotlin/com/micabytes/ink/StoryJson.kt
package com.micabytes.ink


@SuppressWarnings("UtilityClass")
class StoryJson private constructor() {

  init {
    throw AssertionError("StoryJson should never be initialized.")
  }

  companion object {
    const val FILE = "file"
    const val FILES = "files"
    const val CONTENT = "children"
    const val ID = "id"
    const val COUNT = "count"
    const val VARIABLES = "values"
    const val CONTAINER = "container"
    const val COUNTER = "counter"
    const val TEXT = "text"
    const val CHOICES = "choices"
    const val CHOICES_DETAIL = "choice"
    const val IMAGE = "image"
    const val RUNNING = "running"
    const val INDEX = "index"
  }
}
